### PR TITLE
`failure-domain` is no longer used in command line arg.

### PR DESCRIPTION
**Pull request by gyliu513**
> NOTE: Please check the “Allow edits from maintainers” box below to allow 
> reviewers fix problems on your patch and speed up the review process.
> Please delete this note before submitting the pull request.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3352)

### CODE REVIEW

File: docs/concepts/configuration/assign-pod-node.md [hunk 1]
```diff
@@ -183,8 +183,8 @@ flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
 As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
 
 In principle, the `topologyKey` can be any legal label value. However,
-for performance reasons, only a limit set of topology keys are allowed;
-they are specified in the `--failure-domain` command-line argument to the scheduler. By default the allowed topology keys are
+for performance reasons, only a limit set of topology keys are allowed.
```
Comment (chenopis) [inline docs/concepts/configuration/assign-pod-node.md]: "limit" -> "limited"

File: docs/concepts/configuration/assign-pod-node.md [hunk 2]
```diff
@@ -183,8 +183,8 @@ flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
...
+By default the allowed topology keys are
```
Comment (chenopis) [inline docs/concepts/configuration/assign-pod-node.md]: add colon: "...keys are`:`"

File: docs/concepts/configuration/assign-pod-node.md (hunk 1 above)
Comment (gyliu513) [inline docs/concepts/configuration/assign-pod-node.md]: Done

File: docs/concepts/configuration/assign-pod-node.md (hunk 2 above)
Comment (gyliu513) [inline docs/concepts/configuration/assign-pod-node.md]: Done

**Pull request merged**